main.css

```diff
@@ -89,3 +105,25 @@ a {
 	margin: 0 auto;
 	max-width: 800px;
 }
+
+
+/*--------------------------------------
+*	States
+---------------------------------------*/
+
+.is-liked {
+	animation: flash 2s;
+}
+
+/*--------------------------------------
+*	Animation
+---------------------------------------*/
+
+@keyframes flash {
+	from, 50%, to {
+		opacity: 0;
+	}
+	25%, 75% {
+		opacity: 1;
+	}
+}
```

Comment:
https://www.quora.com/Why-is-it-a-common-practice-to-put-a-new-line-at-the-end-of-a-source-code-file